hiss/user/tests/admin_tests/user.py

```diff
@@ -15,3 +15,13 @@ def test_check_in_checks_in_users(self):
         )
         self.user.refresh_from_db()
         self.assertTrue(self.user.checked_in)
+
+    def test_export_emails(self):
```

Comment:
You'll probably want to update this function name to correspond with the updated function name.